src/Cassandra/HostConnectionPool.cs

```diff
@@ -218,6 +218,8 @@ internal void AttemptReconnection()
                 }
                 else
                 {
+                    if (t.Exception != null)
+                        Logger.Error("Error during reconnection attempt", t.Exception);
```

Comment:
Reconnection attempts that failed are not necessarily of log level `Error`: the `ReconnectionPolicy` marks the delay of the reconnection even if its notified as down via events.
We could add the exception string below, something like:
```csharp
Logger.Info("Reconnection attempt to host {0} failed: {1}", _host.Address, t.Exception);
```